uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -391,7 +394,7 @@ sub getObjective
 
 =pod
 
-B<identify_scan_db( C<$center_name>, C<$objective>, C<$fileref>, C<$dbhr> )>
+B<identify_scan_db( C<$center_name>, C<$objective>, C<$fileref>, C<$dbhr>, C<$db> )>
```

Comment:
this will require updating the PerlDoc when that branch is merged, so will add the tag for it.